disks-and-volumes.md

```diff
@@ -73,7 +77,10 @@ spec:
       - name: mypvcdisk
         volumeName: mypvc
         # This makes it a disk
-        disk: {}
```

Comment:
Is there still an example with an empty disk section?

Would be good to keep this example.

Reply:
+1